telehealth-example/README.md

```diff
@@ -0,0 +1,43 @@
+# Telehealth Example
+
+`telehealth-example` is a basic chat application that features communication between doctor and patient.
+
+<img src="../assets/telehealth-example.png" alt="Telehealth example for Android" style="width:300px"/> 
+
+## Prerequisites
+
+This application uses [PubNub Kotlin SDK](https://github.com/pubnub/kotlin) (>= 7.0.0) for chat
+components and [Jetpack Compose](https://developer.android.com/jetpack/compose) as the UI Toolkit.
+
+To use the app, you need:
+
+* [Android Studio](https://developer.android.com/studio/preview) (>= Bumblebee 2021.1.1)
+* PubNub account on [Admin Portal](https://dashboard.pubnub.com/)
+
+## Features
+
+The `telehealth-example` app showcases these components and features:
+
+* [MessageInput](https://www.pubnub.com/docs/chat/components/android/ui-components-android#messageinput)
+* [MessageList](https://www.pubnub.com/docs/chat/components/android/ui-components-android#messagelist)
+* [ChannelList](https://www.pubnub.com/docs/chat/components/android/ui-components-android#channellist)
+* [MessageReactions](https://www.pubnub.com/docs/chat/components/android/message-reactions-android)
+* Message Menu (with the copying option)
+
+## Usage
+
+Read the [Getting Started](https://www.pubnub.com/docs/chat/components/android/get-started-android)
+guide to learn how to use the app and better understand the logic behind it.
+
+## Usernames
+
```

Comment:
You can log into the app either as a doctor or as one of the patients. Type in one of the available usernames and tap **Log in**. You don't need a password to log in.

Reply:
Resolved